update_twiki.py

```diff
@@ -0,0 +1,81 @@
+"""
+Twiki module: Creates a database for missing samples(too be expanded and completed)
+"""
+import sys
+import sqlite3
+import logging
+sys.path.append('/afs/cern.ch/cms/PPD/PdmV/tools/McM/')
+from rest import McM
+#XSDB wrapper
+from request_wrapper import RequestWrapper
+
+#XSDB pycurl requester instance
+xsdb_request = RequestWrapper()
+
+
+# McM instance
+mcm = McM(dev=False, debug=False, cookie='cookie.txt')
+
+# Logger
+logging.basicConfig(format='[%(asctime)s][%(levelname)s] %(message)s', level=logging.INFO)
+logger = logging.getLogger()
+
+def main():
+    conn = sqlite3.connect('twiki.db')
+    c = conn.cursor()
+    # Create table if it does not exist
+    c.execute('''CREATE TABLE IF NOT EXISTS twiki_samples
+                 (prepid text PRIMARY KEY NOT NULL,
+                  dataset text NOT NULL,
+                  extension text,
+                  total_events integer NOT NULL,
+                  campaign text NOT NULL,
+                  resp_group text NOT NULL,
+                  cross_section float NOT NULL,
+                  fraction_negative_weight float NOT NULL,
+                  target_num_events real NOT NULL   )''')
+
+    # Clear the table
+    c.execute('DELETE FROM `twiki_samples`')
+    conn.commit()
+
+    # Get all needed requests
+    twiki_samples_fall_18_candidates = mcm.get('requests', query='member_of_campaign=RunIIFall18*')
+    total_events_threshold = 20000000
+    for twiki_request in twiki_samples_fall_18_candidates:
+        if twiki_request['total_events'] > total_events_threshold:
```

Comment:
I don't know if this is a little mistake or not, but maybe 20M requests should be included too? I.e. `>=` instead of `>`. Not saying this is wrong, just drawing attention to this small detail.

Reply:
I changed the value of the threshold to 0 since we want the complete list of missing samples!